src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,302 @@
-// your components will all go in this `component` directory.
-// feel free to change this component.js into TodoList.js
+import React, { Component } from 'react';
+import './Todo.css';
+import Todo from './Todo'
+
+// created class component with constructor and props to pass attributes
+
+
+class ToDoList extends Component {
+    constructor(props) {
+        super(props);
+
+
+        // set the state equal to an array
+        this.state = {
+            items: []
+        };
+        //  Pomodoro functionality 
+        var pomodoro = {
+            started: false,
+            minutes: 0,
+            seconds: 0,
+            fillerHeight: 0,
+            fillerIncrement: 0,
+            interval: null,
+            minutesDom: null,
+            secondsDom: null,
+            fillerDom: null,
+            init: function () {
+                var self = this;
+                this.minutesDom = document.querySelector('#minutes');
```

Comment:
All of this is query selector stuff is totally unnecessary inside a react component. You can set functions to dom elements inline in the jsx as you have done with everything outside of this pomodoro.